test/transform.cpp

```diff
@@ -86,6 +86,10 @@ void transformTest(string pTestFile, string pHomographyFile, const af_interp_typ
 
     ASSERT_EQ(AF_SUCCESS, af_transform(&outArray, sceneArray, HArray, objDims[0], objDims[1], method, invert));
 
+    int device = -1;
+    ASSERT_EQ(AF_SUCCESS, af_get_device(&device));
+    ASSERT_EQ(AF_SUCCESS, af_sync(device));
+
```

Comment:
I know. It's failing under different circumstances so I wanted to narrow it down. I myself would not want this going into branches.